scripts/entrypoint.sh

```diff
@@ -22,4 +22,26 @@ if [[ $KF_INGEST_APP_MODE = "production" ]]; then
     fi
 fi
 
+INGEST_ERROR=0
+
 kidsfirst "$@"
+
+if [[ $? -ne 0 ]]; then
+    INGEST_ERROR=1
+fi
+
+# During deployment - rm ingest package after ingest completes
+if [[ $CLEANUP ]]; then
+    INGEST_PACKAGE_DIR="/data/packages/$CLEANUP"
+    if [[ -d $INGEST_PACKAGE_DIR ]]; then
+        echo "Start cleanup ..."
+        echo "Deleting ingest package $INGEST_PACKAGE_DIR"
+        rm -rf $INGEST_PACKAGE_DIR
```

Comment:
please quote this variable and put either `set -u` or `set -o nounset` at the start of the file

Reply:
k, will update

Reply:
Also, what if the script wasn't run by docker?

Reply:
Consider...

```
#!/bin/bash

mkdir -p "test"
CLEANUP="foo bar"
touch "test/$CLEANUP"
echo "CLEANING $CLEANUP"
ND="test/$CLEANUP"
rm $ND
echo "done"
```

prints:
> CLEANING foo bar
> rm: test/foo: No such file or directory
> rm: bar: No such file or directory
> done

vs 

```
#!/bin/bash

mkdir -p "test"
CLEANUP="foo bar"
touch "test/$CLEANUP"
echo "CLEANING $CLEANUP"
ND="test/$CLEANUP"
rm "$ND"
echo "done"
```

prints:
> CLEANING foo bar
> done

Reply:
I think that CLEANUP should be passed in as an argument to the script as possible and the check should not be based on whether the environment has something set.